### DISCUSSION TITLE
Skin material B to _Bibo Option

### DISCUSSION THREAD

**Discussion by z32o** [xivdev/Penumbra#153] [Ideas] [answered]
Currently, Penumbra has functionality to convert **Skin Material B to D**, **Skin Material D to B**, **Skin Material A to E**, and **Skin Material E to A**. 

Bibo+, one of the most popular body mods received an update today (2022, Feb. 16th) that adds _bibo to the material slots.

Having a **Skin Material B to _Bibo** option would allow Bibo+ users to directly use older mods that use the mat B slot to the _Bibo slot

**Comment by Ottermandias** [comment, marked as answer]
I will replace the buttons with a general input string to input string function.
Also, this is stupid.
